packages/components/src/types/page.ts

```diff
@@ -124,9 +124,12 @@ export const CollectionPagePageSchema = Type.Intersect([
   }),
 ])
 
-export const ContentPagePageSchema = Type.Object({
-  contentPageHeader: ContentPageHeaderSchema,
-})
+export const ContentPagePageSchema = Type.Composite([
+  Type.Object({
+    contentPageHeader: ContentPageHeaderSchema,
+  }),
+  imageSchemaObject,
+])
```

Comment:
question: hmm was this change intended? Seems unrelated to index pages

Reply:
oh this got removed downstream - i inlined it into the `header` instead, lemme dup the change here